dhis-2/dhis-services/dhis-service-dxf2/src/main/java/org/hisp/dhis/dxf2/expressiondimensionitem/ExpressionDimensionItemService.java

```diff
@@ -101,6 +101,11 @@ else if ( uids.length > 0 )
      */
     public boolean isValidExpressionItems( String expression )
     {
+        if ( expression.chars().allMatch( Character::isDigit ) )
```

Comment:
There's a StringUtils.isNumeric helper in apache commons, we might want to use that